glTFLoader/Interface.cs

```diff
@@ -428,5 +429,289 @@ public static void SaveBinaryModel(this Gltf model, byte[] buffer, BinaryWriter
+                                fileStream.Write(binBufferData, bufferView.ByteOffset, bufferView.ByteLength);
+                                bufferView.ByteOffset = (int)fileStreamPosition;
+                            }
+
+                            var newIndex = bufferViews.Count;
+                            if (index != newIndex)
+                            {
+                                indexMap.Add(index, newIndex);
+                            }
+
+                            bufferViews.Add(bufferView);
+                        }
+                    }
+
+                    binByteLength = (int)fileStream.Length;
+                }
+
+                model.BufferViews = bufferViews.ToArray();
+
+                if (binByteLength == 0)
+                {
+                    File.Delete(binFilePath);
+                    if (binBuffer != null)
+                    {
+                        model.Buffers = model.Buffers.Skip(1).ToArray();
+                        foreach (var bufferView in model.BufferViews)
+                        {
+                            bufferView.Buffer--;
+                        }
+                    }
+                }
+                else
+                {
+                    binBuffer.Uri = binFileName;
+                    binBuffer.ByteLength = binByteLength;
+                }
+
+                if (model.Accessors != null)
+                {
+                    foreach (var accessor in model.Accessors)
+                    {
+                        if (accessor.BufferView.HasValue)
+                        {
+                            if (indexMap.TryGetValue(accessor.BufferView.Value, out int newIndex))
+                            {
+                                accessor.BufferView = newIndex;
+                            }
+                        }
+                    }
+                }
+            }
+
+            if (model.Buffers != null)
+            {
+                for (var index = 1; index < model.Buffers.Length; index++)
+                {
+                    var buffer = model.Buffers[index];
+                    if (!buffer.Uri.StartsWith("data:"))
+                    {
+                        var sourceFilePath = Path.Combine(inputDirectoryPath, buffer.Uri);
+                        var fileName = $"{inputFileName}{index}.bin";
+
+                        if (File.Exists(sourceFilePath))
+                        {
+                            var destinationFilePath = Path.Combine(outputDirectoryPath, fileName);
+                            File.Copy(sourceFilePath, destinationFilePath, true);
+                        }
+
+                        buffer.Uri = fileName;
+                    }
+                }
+            }
+
+            Interface.SaveModel(model, Path.Combine(outputDirectoryPath, $"{inputFileName}.gltf"));
+        }
+        private static Image.MimeTypeEnum? GetMimeType(string uri)

```

Comment:
nit: Missing a newline here.

```suggestion

        private static Image.MimeTypeEnum? GetMimeType(string uri)
```